packages/common/src/hooks/useSortBy/useSortBy.ts

```diff
@@ -1,8 +1,9 @@
 import { ObjectWithStringKeys } from './../../types/utility';
 import { useCallback, useState } from 'react';
+import { GenericColumnKey } from '../..';
 
 export interface SortRule<T extends ObjectWithStringKeys> {
-  key: keyof T;
+  key: keyof T | GenericColumnKey;
```

Comment:
sorting by the generic columns? I expect I'll understand why soon enough

Reply:
uhh yeah, sorry. I wanted to push the branch up and this was making TS happy.. will fix!